src/services/recim/activity.ts

```diff
@@ -4,58 +4,39 @@ import { Team } from './team';
 import { Series } from './series';
 import { Lookup } from './recim';
 
-export type Activity = {
-  ID: number;
+type BaseActivity = {
   Name: string;
+  StartDate: string;
+  EndDate: string;
   RegistrationStart: string;
   RegistrationEnd: string;
-  RegistrationOpen: boolean;
   SportID: Sport;
   TypeID: number;
   MinCapacity: number;
   MaxCapacity: number;
   SoloRegistration: boolean;
+};
+
+export type Activity = BaseActivity & {
+  ID: number;
+  RegistrationOpen: boolean;
   Logo: string;
   Series: Series[];
   Team: Team[];
 };
 
-type CreatedActivity = {
+type CreatedActivity = BaseActivity & {
   ID: number;
```

Comment:
I don't mind this logic if it works, but if you start it here make sure you handle it in the other service files eventually!